Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Buttons/Krypton.Toolkit.Suite.Extended.Buttons 2019.csproj

```diff
@@ -77,4 +77,9 @@
         <LastGenOutput>Resources.Designer.cs</LastGenOutput>
       </EmbeddedResource>
     </ItemGroup>
+  
+    <!--TODO: Needs looking at, attemping to add the build output to a ZIP file for distribution-->
+    <!--<Target Name="PostBuild" AfterTargets="PostBuildEvent">
+      <Exec Command="if $(ConfigurationName) == Debug (powershell -Command Get-ChildItem -Path '$(TargetDir)publish' -Recurse|Compress-Archive -DestinationPath '$(SolutionDir)PublishOutput$(ProjectName)-$(ConfigurationName).zip' -Force)" />
```

Comment:
If it is for distribution then this should state "Release"

Reply:
Fixed